src/types.ts

```diff
@@ -1,37 +1,53 @@
+import { WebpackError } from 'webpack'
+
+export type FunctionWithErrors = (errors?:WebpackError[]) => void
+export type OnError = 'skip' | 'execute'
```

Comment:
Please change the parameter to bool, doing string with 2 parameters is very strange. 
```
OnErrorSkip = boolean
```